engine/config_integrations/webhook.py

```diff
@@ -32,7 +32,18 @@
 
 web_image_url = slack_image_url
 
-sms_title = web_title
+sms_title = """\
```

Comment:
I propose to keep this closer to the current version of SMS (probably can drop the alert count, @Matvey-Kuk wdyt?), for example:
```python
"""\
You are invited to check an alert group #{{ grafana_oncall_incident_id }}
Organization: {{ grafana_oncall_organization }}
Integration: {{ integration_name }}
Title: {{ this could be different based on integration }}
Link: {{ grafana_oncall_slack_link or grafana_oncall_link }}

Your Grafana OnCall <3
"""
```

Probably would be a good idea to add `grafana_oncall_slack_link` to jinja2 context as well, so the there's a link to Slack message as it is currently.

Also please note that there are multiple integrations in [engine/config_integrations](https://github.com/grafana/oncall/tree/dev/engine/config_integrations), so we must make sure to set sms_title on all of them so it can be used not only for webhook integration, but others as well (e.g. alertmanager)

Reply:
I like seeing `grafana_oncall_link` link here. We want to make it les slack-specific. Also nice to add `grafana_oncall_link_slack` as @vadimkerr mentioned.